addons/website/static/tests/tours/media_dialog.js

```diff
@@ -203,3 +204,84 @@ registerWebsitePreviewTour("website_media_dialog_insert_media", {
         trigger: ":iframe .s_text_block p > span.fa",
     },
 ]);
+
+registerWebsitePreviewTour("website_replace_remove_image", {
+    url: "/",
+    edition: true,
+}, () => [
+    ...insertSnippet({
+        id: "s_text_image",
+        name: "Text - Image",
+        groupName: "Content",
+    }),
+    {
+        content: "Open the media dialog from the snippet",
+        trigger: ":iframe .s_text_image img",
+        run: "dblclick",
+    },
+    {
+        content: "Click on the toolbar's 'Add URL' button",
+        trigger: ".o_upload_media_url_button",
+        run: "click",
+    },
+    {
+        content: "Edit input field value",
+        trigger: ".o_we_url_input",
+        run: "edit website/static/src/img/backgrounds/city.jpg",
+    },
+    {
+        content: "Click on the toolbar's 'Add URL' button",
+        trigger: ".o_upload_media_url_button",
+        run: "click",
+    },
+    {
+        content: "Open the media dialog from the snippet",
+        trigger: ":iframe .s_text_image img",
+        run: "dblclick",
+    },
+    {
+        content: "Click on remove attachment",
+        trigger: ".o_we_attachment_selected",
+        run: function () {
+            this.anchor.querySelector(".fa-trash").click();
+        },
+    },
+    {
+        content: "Confirm the removal of the attachment",
+        trigger: ".btn:contains('Ok')",
+        run: "click",
+    },
+    {
+        content: "Click 'Discard' to close the media dialog",
+        trigger: ".o_select_media_dialog .btn:contains(Discard)",
+        run: "click",
+    },
+    ...clickOnSave(),
+    {
+        content: "Verify that the image is replace with thumbnail and able to save website",
```

Comment:
"Ensure the image is replaced with a placeholder thumbnail"

and do we need to run: "click" to verify whether image is replaced ?